tests/tools/importexport/orm/test_extras.py

```diff
@@ -89,7 +89,7 @@ def test_absence_of_extras(self):
             self.imported_node.get_extra('c')
 
     def test_extras_import_mode_keep_existing(self):
-        """Check if old extras are not modified in case of name collision"""
+        """Check if old extras are not modified in case of name collision (keep org, create new, leave org)"""
```

Comment:
what does this mean?
what is org?

Reply:
original. This was to make the description of @yakutovicha's tests more explicit. Will make them even more explicit :)